package-lock.json

```diff
@@ -7006,9 +7006,9 @@
       }
     },
     "merge": {
-      "version": "1.2.0",
-      "resolved": "https://registry.npmjs.org/merge/-/merge-1.2.0.tgz",
-      "integrity": "sha1-dTHjnUlJwoGma4xabgJl6LBYlNo="
+      "version": "1.2.1",
+      "resolved": "https://registry.npmjs.org/merge/-/merge-1.2.1.tgz",
+      "integrity": "sha512-VjFo4P5Whtj4vsLzsYBu5ayHhoHJ0UqNm7ibvShmbmoz7tGi0vXaoJbGdB+GmDMLUdg8DpQXEIeVDAe8MaABvQ=="
```

Comment:
Mildly confused by this.
I don't see how this was fixed, exactly.
Shouldn't we be updating dependency versions in package.json to resolve vuln **merge** versions? Wouldn't this just revert if somebody clobbered the package-lock.json file and regenerated it (to fix a merge conflict, etc)?

Shouldn't we just bump **jest** to the latest version?

Reply:
Yeah I'm not sure how this fixed it either ... I just kept doing what `npm audit` told me to do until it passed.